packages/@ember/-internals/glimmer/lib/modifiers/custom.ts

```diff
@@ -109,18 +126,36 @@ class InteractiveCustomModifierManager<ModifierInstance>
     return new CustomModifierState(element, definition.delegate, instance, capturedArgs);
   }
 
-  getTag({ args }: CustomModifierState<ModifierInstance>): Tag {
-    return args.tag;
+  getTag({ args, tag }: CustomModifierState<ModifierInstance>): Tag {
+    return combine([tag, args.tag]);
   }
 
   install(state: CustomModifierState<ModifierInstance>) {
-    let { element, args, delegate, modifier } = state;
-    delegate.installModifier(modifier, element, args.value());
+    let { element, args, delegate, modifier, tag } = state;
+
+    let tracked = track(() => {
```

Comment:
I would think this should be guarded as well?

```js
let { capabilities } = delegate;
let shouldAutoTrack = capabilities === undefined || capabilities.disableLifecycleTracking !== true;

let tracked;

if (shouldAutoTrack) {
  tracked = track(() => {
    delegate.installModifier(modifier, element, args.value());
  });
} else {
  delegate.installModifier(modifier, element, args.value());
}

if (tracked !== undefined) {
  update(tag, tracked);
}
```

Reply:
Wrapping it in a `track` call prevents external `track`s from watching the values. My logic was that if you disabled tracking, you probably don't want _anything_ to be able to track these computes, so we should swallow any autotracking within and throw it away. We can use the `untrack` function for this though.

Reply:
> you probably don't want anything to be able to track these computes, so we should swallow any autotracking within and throw it away

Hmm, I see. I think both are valid paths forward, definitely up to you here. Though if you don't mind, could you add a small inline comment about throwing away the tracker in the case where you have explicitly opted out?